api/pkg/apis/v1alpha1/managers/activations/activations-cleanup-manager.go

```diff
@@ -19,13 +17,13 @@ import (
 )
 
 const (
-	// DefaultRetentionInMinutes is the default time to cleanup completed activations
-	DefaultRetentionInMinutes = 1440
+	// DefaultRetentionDuration is the default time to cleanup completed activations
+	DefaultRetentionDuration = time.Duration(180) * time.Hour * 24
```

Comment:
No need to use time.Duration(180) here. You can just use 180 * 24 * time.Hour.

Reply:
fixed